java/arcs/core/storage/DirectStore.kt

```diff
@@ -41,7 +42,8 @@ class DirectStore<Data : CrdtData, Op : CrdtOperation, T> /* internal */ constru
     val localModel: CrdtModel<Data, Op, T>,
     /* internal */
     val driver: Driver<Data>
-) : ActiveStore<Data, Op, T>(options) {
+) : ActiveStore<Data, Op, T>(options),
```

Comment:
The changes in this file appear to be the crux of any functionality changes in this PR, I will defer to @galganif and @jasonwyatt but we should review this closely to understand any behavior changes or Arcs Storage design implications.

Reply:
if we do not commit #5204, do we still need this?

Reply:
Yes, the coming / latest report will cover four sorts of latency metrics:
1) store/write round-trip
2) fetch/read round-trip
3) setup time (cover the very first synchronization)
4) write-to-read trip (how long/fast a listener/reader can get the update)

The PR helps the 4) a lot regardless of #5204.